libcnb-test/README.md

```diff
@@ -15,35 +15,36 @@ Please use the same tag for feature requests.
 
 ```rust,no_run
 // In $CRATE_ROOT/tests/integration_test.rs
-use libcnb_test::{IntegrationTest, BuildpackReference, assert_contains};
+use libcnb_test::{assert_contains, BuildpackReference, TestRunner, TestConfig};
 
 #[test]
 fn test() {
-    IntegrationTest::new("heroku/builder:22", "test-fixtures/app")
-        .buildpacks(vec![
-            BuildpackReference::Other(String::from("heroku/openjdk")),
```

Comment:
(No action needed) I wonder if this README example of using `.buildpacks()` is the reason why some of the other projects had been using them unnecessarily (when only testing the current buildpack)? If so, I think removing the `.buildpacks()` usage here (as you have done) is a good idea, to keep things simple / avoid unnecessary usage :-)